awx/main/migrations/0203_remove_team_of_teams.py

```diff
@@ -0,0 +1,64 @@
+import logging
+
+from django.db import migrations
+
+logger = logging.getLogger('awx.main.migrations')
+
+
+def consolidate_indirect_user_roles(apps, schema_editor):
+    """
+    A user should have a member role for every team they were indirectly
+    a member of. ex. Team A is a member of Team B. All users in Team A
+    previously were only members of Team A. They should now be members of
+    Team A and Team B.
+    """
+
+    # get object roles for membership on teams
+    ObjectRole = apps.get_model('dab_rbac', 'ObjectRole')
```

Comment:
There's another model from the old API, the `main.Role` model. This is synchronized with thew new models (this model) via signals, but signals don't run in migrations. So the simplest solution might be to mirror your changes in both models.

Reply:
figuring out the two failing tests and from the wording would this test actually need to be removed since it's testing giving a team read permissions to another team or is that still acceptable? https://github.com/ansible/awx/blob/devel/awx/main/tests/functional/dab_rbac/test_translation_layer.py#L177-L187

Reply:
second test looks like another case of a team on team 
 expectation. that should fail I would think though not sure why they're both failing now. maybe because of the setting addition now? https://github.com/ansible/awx/blob/devel/awx/main/tests/functional/test_fixture_factories.py#L47-L59. might be useful to pull out the team on teams part and make that an expected fail test?